docs/guides/themeable-to-emotion-migration-guide.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Themeable to Emotion Migration Guide for Version 8.0
+category: Guides
+order: 5
+---
+
+## Themeable to Emotion Migration Guide for Version 8.0
+
+The main feature of the Version 8.0 release is the switch of the theming engine. We replaced the in-house `ui-themeable` theming solution with the popular [emotion.js](https://emotion.sh/) library.
```

Comment:
I'd add the following: "We removed Themable and its supporting packages from InstUI, all the components in 8.0 use emotion.js."